test-suites/_gateway_punk.spec.ts

```diff
@@ -418,3 +381,232 @@ describe("Punk nToken Mint and Burn Event Accounting", () => {
     ).to.equal(userBalance.add(1));
   });
 });
+
+const fixture = async () => {
```

Comment:
only minor comment would be to better place the fixture inside the describe() clause it belongs too (line 412)

Reply:
updated   @ivansolomonoff